drivers/sensor/hp206c/hp206c.c

```diff
@@ -288,9 +287,10 @@ static int hp206c_init(const struct device *dev)
 {
 	struct hp206c_device_data *hp206c = dev->data;
 
-	hp206c->i2c = device_get_binding(DT_INST_BUS_LABEL(0));
-	if (!hp206c->i2c) {
-		LOG_ERR("I2C master controller not found!");
+	hp206c->i2c = DEVICE_DT_GET(DT_INST_BUS(0));
+
+	if (!device_is_ready(hp206c->i2c)) {
```

Comment:
we should be moving i2c to driver's config instead (using i2c_dt_spec), so that it gets stored in ROM

Reply:
Done!